examples/samples/KOSMAPOSL_hkem_spect.par

```diff
@@ -0,0 +1,108 @@
+KOSMAPOSLParameters :=
```

Comment:
Should be highlighted differently in CHANGES.md (i.e., added HKEM SPECT example parameter file)